### ISSUE TITLE
outred.org isnt even set up?

### ISSUE THREAD

**Issue by CycloKid** [OutRed/outredgames-v3.5#1]
this right down below this text
![NbbzMgR8Ee6UzgJCrBEAAg](https://github.com/OutRed/outredgames-v3.5/assets/63879422/a268a590-4480-4f9f-ad0c-6f491e9a8201)
idk maybe put an "under construction" page or specify it in the repository? just a suggestion

**Comment by OutRed** [comment]
I’m fixing it, it’s just the GIT integration is being weird. It’ll probably be fixed today.

**Comment by CycloKid** [comment]
ah, makes sense. gui-based website services are sometimes more annoying than efficient

**Comment by OutRed** [comment]
Yeah. This new one is really annoying, but it’s cheap and offers exactly what I need.
